.gitignore

```diff
@@ -15,5 +15,10 @@
 # vendor/
 
 # Local configuration
-fixtures/
+fixtures/backups
+fixtures/certs
+fixtures/creds
+fixtures/db
+fixtures/db.old
```

Comment:
I agree that we should revert this to its previous state after merging; I feel like it's easier to maintain globally ignored directories with specific exceptions. I think `!fixtures/datagen` would not ignore the datagen file in fixtures, even if `fixtures/` was ignored, though I'm not 100% sure about the syntax. We could also use `git add -f fixtures/myfile` to force add a file even if it is in .gitignore, if that's easier.